react/features/base/dialog/actions.js

```diff
@@ -53,7 +55,7 @@ export function openDialog(component: Object, componentProps: ?Object) {
  * @returns {Function}
  */
 export function toggleDialog(component: Object, componentProps: ?Object) {
-    return (dispatch: Dispatch, getState: Function) => {
+    return (dispatch: Dispatch<*>, getState: Function) => {
```

Comment:
Why doesn't this need to be `Dispatch<any>`?

Reply:
Oh, looks my blanket search / replace failed there. I'll fix it, thanks!